hare/broker.go

```diff
@@ -145,6 +145,7 @@ func (broker *Broker) eventLoop() {
 		case task := <-broker.tasks:
 			task()
 		case <-broker.CloseChannel():
+			log.Info("Broker exiting")
```

Comment:
Is it really info?

Reply:
It could be warn instead. Of course, if it suddenly getting closed it is not expected. I'll change it to warn.